.gitlab-ci.yml

```diff
@@ -97,10 +114,11 @@ test:smoketests:linux:
   image: golang:1.18-buster
   before_script:
     - apt-get update && apt-get install -q -y make liblzma-dev libssl-dev
-    - bash tests/test_sign_with_hsm/test_sign_with_hsm.sh --setup
+    - ./tests/test_sign_with_hsm/test_sign_with_hsm.sh --setup
+    - export TEST_MENDER_ARTIFACT_PATH=./mender-artifact-linux
```

Comment:
Question: Is this just done for the aesthetics?

Reply:
yes. It some kind of logical to setup everything is needed in `before_script` and then in `script` just execute the test. And shebang is present in the script, so, `bash` is not needed.

Reply:
It is usually considered good commit hygiene to seperate functionality from cleanups and misc other fixes.

So if this is not strictly part of the PKCS functionality, can you split it out?

Reply:
👍